libconsensus/pbft/PBFTEngine.h

```diff
@@ -484,6 +490,19 @@ class PBFTEngine : public ConsensusEngineBase
         return boost::filesystem::space(path).available > 1024;
     }
 
+    void updateViewMap(IDXTYPE const& idx, VIEWTYPE const& view)
+    {
+        WriteGuard l(x_viewMap);
+        if (idx != nodeIdx())

```

Comment:
Why do it need to check the value of idx? I think it is ok to just set m_viewMap[idx] as view

Reply:
done.